specs/method-identifiers.html

```diff
@@ -305,7 +305,7 @@
         <p>This section relies on several other underlying specifications.</p>
         <dl>
           <dt>URL</dt>
-          <dd>The terms <em>URL</em>, <em>absolute URL</em>, <em>URL parser</em>, and <em>URL equivalence</em> are defined by [[!url-1-20141209]] <small><em>(or the editor's draft)</em></small>.</dd>
+          <dd>The terms <a>URL</a>, <a>absolute URL</a>, <a>URL parser</a>, and <a>URL equivalence</a> are defined by [[!url-1-20141209]] <small><em>(or the editor's draft)</em></small>.</dd>
         </dl>
```

Comment:
> Should these <a> tags not be <dfn> tags?

Not if they are terms define in other specs, right? As such, I think Respec automatically resolves the `\<a>`-marked references to generate hyperlinks that lead to the external definitions in the correct specs.